### PR TITLE
fixing issue with _parent mapping and ES 2.3.3

### PR DESCRIPTION
**Pull request by oleg-andreyev**
This will fix issue with `Mapping definition for [_parent] has unsupported parameters:  [identifier : id] [property : null]`

### CODE REVIEW

File: DependencyInjection/FOSElasticaExtension.php [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 namespace FOS\ElasticaBundle\DependencyInjection;
 
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
```
Comment (solivier) [inline DependencyInjection/FOSElasticaExtension.php]: Why did you remove this line ? It seems used in this class

Comment (oleg-andreyev) [inline DependencyInjection/FOSElasticaExtension.php]: oops, it seems phpStorm had "Optimize imports" option enabled, reverted this part.

**Comment by XWB** [comment]
Thanks.

**Comment by oleg-andreyev** [comment]
Awesome! Thanks!
